cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/async/OperationFuture.java

```diff
@@ -101,4 +103,12 @@ public String progressString() {
   public OperationProgress operationProgress() {
      return _operationProgress;
   }
+
+  public void setFinishTime(long finishTime) {
+    _finishTime = finishTime;
+  }
+
+  public long getFinishTime() {
```

Comment:
In getters, we typically do not use `get` in the method name -- i.e. `getFinishTime` -> `finishTime`.